app/code/Magento/Sales/Block/Adminhtml/Order/Create/Form.php

```diff
@@ -135,7 +135,7 @@ public function getStoreSelectorDisplay()
     {
         $storeId = $this->getStoreId();
         $customerId = $this->getCustomerId();
-        if ($customerId !== null && !$storeId) {
+        if (!empty($customerId) && !$storeId) {
```

Comment:
I would recommend simplifying this condition to:
```
if ($customerId && !$storeId) {
```

Reply:
I have refactored the if condition